tensorflow_datasets/core/holder.py

```diff
@@ -0,0 +1,295 @@
+import os
+import re
+import tarfile
+
+import tensorflow as tf
+from tensorflow_datasets.core.utils import py_utils
+from tensorflow_datasets.core import lazy_imports
+from tensorflow_datasets.core import naming
+from tensorflow_datasets.scripts import create_new_dataset
+import json
+import shutil
+import zipfile
+
+
+# TODO if download 2 files return array and download both
+
+NUMBER_OF_CREATED_FILE = 0
+
+class Holder(object):
```

Comment:
Could you please add docstring comments on classes and public methods?

Just a one line description of what this is and what it's for.
This helps reading the code.